package.json

```diff
@@ -76,8 +80,6 @@
     "url": "https://github.com/dilvie/credential/issues"
   },
   "pre-commit": [
-    "lint",
-    "validate",
-    "test"
+    "validate"
```

Comment:
Change this to lint. We want `pre-commit` to be really fast. Travis-CI will catch the rest. =)